lib/plugins/aws/lib/validate.js

```diff
@@ -1,19 +1,14 @@
 'use strict';
 
-const BbPromise = require('bluebird');
-
 module.exports = {
-  validate() {
+  async validate() {
```

Comment:
This one ideally should be kept as _sync_ function, there's no _async_ logic in it, it shouldn't return promise.

Reply:
This caused some cascading test failures and so more changes. I've split the changes for validate into a separate commit to hopefully make it easier to review.